typescript/example/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Handler } from "./index.wing";
+
+export const handler: Handler = async ({ myBucket, myQueue }, event: any) => {
```

Comment:
Other than the IDE, would this be type-checked at any point during compilation?

Reply:
At the moment, no. We just bundle with esbuild. I guess we can add a typecheck pass. I'll add a TODO